### ISSUE TITLE
Trouble Launching 8.1 under Windows 11

### ISSUE THREAD

**Issue by mcculleymike** [thargor6/JWildfire#430]
Get error message: No JRE found in "/C:/Users/mccul/OneDrive/Downloads/j-wildfire-8.10/j-wildfire-launcher.jar"
and: " Attempting to launch JWildfire using the command:
cmd /C start /low /b "JWildfire" "C:\Program Files\Java\jre1.8.0_341\bin\java" -Xms256m -Xmx4024m -Dsun.java2d.uiScale=12.0 -Djava.library.path=C:\Users\mccul\OneDrive\Downloads\j-wildfire-8.10\lib\x64 -cp C:\Users\mccul\OneDrive\Downloads\j-wildfire-8.10\lib\j-wildfire.jar org.jwildfire.swing.JWildfire"

What is the JRE? I added the runtime in the initial Launcher. Added the path to my Java folder, and selected Jave.exe. Do I have to move the runtime inside the J-Wildfire folder, which is what this error sounds like? Long-time user confused, never remember having to do that, the link in the setup always seemed to work.
Mike McCulley

**Comment by mcculleymike** [comment]
<img width="294" alt="Screenshot 2023-05-22 142438" src="https://github.com/thargor6/JWildfire/assets/3164697/4485f142-634d-4248-a538-622ef6b64b11">
I put it on the hard drive, but still no luck. I did notice on my command bar on the bottom I had several icons for java. Apparently it is showing<img width="294" alt="Screenshot 2023-05-22 142438" src="https://github.com/thargor6/JWildfire/assets/3164697/b33572c5-7ab2-49b1-b5d8-c3bab06a03d4">pened a java instance and loaded a file into the editor. I am attaching a screenshot of one of those icons. When I click on the icon nothing happens. I also changed the UI scale to 8. Is that still too big? thanks, Mike

**Comment by thargor6** [comment]
UI scale is a multipülicator. So, when you have a window with width 1000 pixels, it will be 8000 pixels wide. So, you will need a huge screen and a lot of mermory (because the window is 64 times as large as with UI scale 1.0).

**Comment by mcculleymike** [comment]
I made the changes to having it all load from my hard drive instead of Onedrive, which is a great idea, but unfortunately didn't fix it. I think the java window is being blocked as a popup by Windows Defender. As I mentioned, I just upgraded from Windows 10 to Windows 11. I think Defender is blocking it because I can see a window frame pop up and exit immediately, and nothing happens except . . . everytime I run the JWlauncher a Java icon is on my command bar on the bottom. I can tell it tried to load your program because of 2 windows it shows. One trying to load a fractal into the editor, and the second one is "help question of the day". I am trying to find an exceptions Window in Microsoft Defender to allow Java and/or JW.exe to be allowed to run as a popup. I'll let you know if I make any progress. thanks, Mike

JFrom: Andreas Maschke ***@***.***>
Sent: Monday, May 22, 2023 3:37 PM
To: thargor6/JWildfire ***@***.***>
Cc: mcculleymike ***@***.***>; Author ***@***.***>
Subject: Re: [thargor6/JWildfire] Trouble Launching 8.1 under Windows 11 (Issue #430)


UI scale is a multipülicator. So, when you have a window with width 1000 pixels, it will be 8000 pixels wide. So, you will need a huge screen and a lot of mermory (because the window is 64 times as large as with UI scale 1.0).
